### DISCUSSION TITLE
Helm Docs

### DISCUSSION THREAD

**Discussion by robrankin** [jertel/elastalert2#241] [Ideas]
The Helm related docs seem to be pretty... off.  The chain of links starts on GH, traverses readthedocs.io, then dockerhub, and then back to GH only to 404.

* Repo README.md
* Links to : https://elastalert2.readthedocs.io/en/latest/running_elastalert.html
* https://elastalert2.readthedocs.io/en/latest/running_elastalert.html#as-a-kubernetes-deployment
*"See the instructions provided on Github for more information on how to install, configure, and run the chart."
* Actually links to Dockerhub, not Guthub: https://hub.docker.com/r/jertel/elastalert2
* Kubernetes section links to : https://github.com/jertel/elastalert2/chart/elastalert2/README.md
* "Not Found"

**Comment by ferozsalam** [comment]
Thanks @robrankin, this is my fault! Will fix now.

**Reply by ferozsalam** [reply]
I have opened #242 for the broken link in readthedocs.

For the 'Kubernetes' section with the broken link on Docker Hub, it looks like an outdated copy of the Github project README. I'm not sure if this is something that Docker Hub caches and refreshes periodically, or something that @jertel manually updates. In either case, refreshing with the current README will resolve that issue, but I don't have permissions to do that.

**Reply by jertel** [reply]
I've merged your PR @ferozsalam and also made a quick adjustment on the README. This should avoid sending readers to the readthedocs.io site if they are actually interested in the Helm chart details. The Docker hub readme has also been removed and replaced with three links: GitHub README, ReadTheDocs page, and the Helm Chart details.

Thanks for the help.
